core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -374,6 +373,26 @@ public String toString() {
     return "Alluxio master @" + mRpcConnectAddress;
   }
 
+  private static void validateRPCExecutor() {
+    int parallelism = ServerConfiguration.getInt(PropertyKey.MASTER_RPC_EXECUTOR_PARALLELISM);
+    Preconditions.checkArgument(parallelism > 0,
+            String.format("Cannot start Alluxio master gRPC thread pool with %s=%s! "
+                            + "The parallelism must be greater than 0!",
+                    PropertyKey.MASTER_RPC_EXECUTOR_PARALLELISM.toString(), parallelism));
```

Comment:
I don't fully recall, but what does this `toString` return? If it returns a lot of other information, then we could use something like `PropertyKey.Name.MASTER_RPC_EXECUTOR_PARALLELISM`?

(this applies to other `toString()` calls in this file)

Reply:
The `toString()` for PropertyKey just returns its name, which is exactly `PropertyKey.Name.MASTER_RPC_EXECUTOR_PARALLELISM` when this PropertyKey is created by the Builder. See the sample exception output in the description of this PR.